README.md

```diff
@@ -11,15 +11,15 @@ to provide a minimalist PSR-7 middleware framework for PHP, with the following
 features:
 
 - Routing. Choose your own router; we support:
-  - [Aura.Router](https://github.com/auraphp/Aura.Router)
-  - [FastRoute](https://github.com/nikic/FastRoute)
-  - [ZF2's MVC router](https://github.com/zendframework/zend-mvc)
+    - [Aura.Router](https://github.com/auraphp/Aura.Router)
+    - [FastRoute](https://github.com/nikic/FastRoute)
+    - [ZF2's MVC router](https://github.com/zendframework/zend-mvc)
 - DI Containers, via [container-interop](https://github.com/container-interop/container-interop).
   Middleware matched via routing is retrieved from the composed container.
 - Optionally, templating. We support:
-  - [Plates](http://platesphp.com/)
-  - [Twig](http://twig.sensiolabs.org/)
-  - [ZF2's PhpRenderer](https://github.com/zendframework/zend-view)
+    - [Plates](http://platesphp.com/)
+    - [Twig](http://twig.sensiolabs.org/)
+    - [ZF2's PhpRenderer](https://github.com/zendframework/zend-view)
```

Comment:
I added indentation on the second-level items above in hopes that RTD will properly indent them.